index.bs

```diff
@@ -606,6 +701,39 @@ SessionResult = (StatusResult)
 
 ```
 
+<div algorithm>
+
+To <dfn lt="updating the event map">update the event map</dfn>, given a |list of
```

Comment:
Having "list" in the variable name feels new and scary to me :) https://fetch.spec.whatwg.org/#convert-header-names-to-a-sorted-lowercase-set is more in the style I'm accustomed to.